src/pyosmeta/cli/update_review_contribs.py

```diff
@@ -0,0 +1,143 @@
+# update the file itself (how often??) so it's worth considering a way to
+# just update the current file and flag entries that we do not want to update
+# TODO: add a key in the contributor_type list for submitting-author
+# TODO: make sure we can add a 3rd or 4th reviewer - crowsetta has this as
+# will biocypher
+# TODO: make sure to add a current editor boolean to the current editors and
+# emeritus ones.
+
+import pickle
+from typing import Dict, List, Optional, Tuple, Union
+
+from pyosmeta.contrib_review_meta import UpdateReviewMeta
+from pyosmeta.file_io import get_api_token
+
+
+def get_clean_user(username: str):
+    return username.lower().strip()
+
+
+def main():
+    GITHUB_TOKEN = get_api_token()
+    updateContribs = UpdateReviewMeta(GITHUB_TOKEN)
+
+    # Two pickle files are outputs of the two other scripts
+    # use that data to avoid having to hit the API again to retrieve.
+    with open("all_contribs.pickle", "rb") as f:
+        contribs = pickle.load(f)
+
+    # Open packages yaml created by running parse_review_issues.py
+    with open("all_reviews.pickle", "rb") as f:
+        packages = pickle.load(f)
+
+    contrib_types = updateContribs.contrib_types
+
+    # TODO: ivan did an if/ else to treat roles with 1 entry person vs multiple
+    # if maintainers, ..
+    # TODO - ?create a class for person types??
+
+    for pkg_name, issue_meta in packages.items():
+        print("Processing review team for:", pkg_name)
+        for issue_review_role in contrib_types.keys():
+            if issue_review_role == "all_current_maintainers":
+                if issue_review_role in issue_meta:
+                    # Loop through each maintainer in the list
+                    for i, a_maintainer in enumerate(issue_meta.get(issue_review_role)):
+                        gh_user = get_clean_user(a_maintainer["github_username"])
+
+                        # TODO: The steps below are almost identical to the reviewers
+                        # Could these alias' be defined only once?
+                        if gh_user not in contribs.keys():
+                            contribs.update(
+                                updateContribs.check_add_user(gh_user, contribs)
+                            )
+                        # contribs_user = contribs[gh_user]
+                        existing_contribs = contribs[gh_user]["contributor_type"]
+                        # TODO: rename?? This is the review_role_yml_key
+                        contrib_key_yml = contrib_types[issue_review_role][0]
+                        contrib_key = contribs[gh_user][contrib_key_yml]
+                        # Update contrib roles
+                        review_role = contrib_types[issue_review_role][1]
+
+                        # Update contrib packages for peer review
+                        contribs[gh_user][
+                            contrib_key_yml
+                        ] = updateContribs.update_contrib_list(contrib_key, pkg_name)
+
```

Comment:
Question: how to use aliases so they update a dictionary or should i use them this way?

how do alias' behave within functions??